core/core.go

```diff
@@ -306,34 +306,34 @@ type SinkSession[
 	I BlockId,
 	F Flags,
 ] struct {
-	accumulator  StatusAccumulator[I]
-	orchestrator Orchestrator[F]
-	store        BlockStore[I]
-	pending      *util.SynchronizedDeque[Block[I]]
-	stats        stats.Stats
+	statusAccumulator StatusAccumulator[I]
```

Comment:
We only have status accumulators, so maybe this isn't necessary, but it keeps things clearer for new comers in my opinion.